lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -500,19 +500,30 @@ use strict;
 use warnings;
 use File::ShareDir ':ALL';
 use Path::Class;
-
-my $abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+use File::Which;
+use {{ $mod }};
+
+my $abs;
+if({{ $mod }}->install_type ne 'system') {
+	$abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+} else {
+	my @opts = which '{{ $bin }}';
+	$abs = $opts[1];
```

Comment:
There are lots of cases where this won’t work.  Better would be to not install the bin wrapper for a system install but that doesn’t handle the case of when the Alien is re-installed as system.

Reply:
I think this might be OK for now. Perhaps we can assume if somebody is reinstalling an Alien, they know what they are doing.

If the alternative is to not install the bin wrapper with a system install, the code should perhaps `die` with instructions on how to remove the wrapper.

Reply:
I agree that either is better than the current state of affairs.   I think I would slightly prefer not installing wrappers on system install and your suggestion of dying for a replacement install where the wrapper was already installed.

Reply:
> I agree that either is better than the current state of affairs. I think I would slightly prefer not installing wrappers on system install and your suggestion of dying for a replacement install where the wrapper was already installed.

How would I do that though?

Reply:
@zmughal I think this may be relying on `[ExecDir]` which is entirely static?